static/scripts/grml-config.sh

```diff
@@ -12,14 +12,16 @@ if command -v wget >/dev/null 2>&1 ; then
   wget -O .zshenv     https://grml.org/console/zshenv
   # optional:
   # wget -O .zshrc.local https://grml.org/console/zshrc.local
+  # wget -O .screenrc https://grml.org/console/screenrc.grml
```

Comment:
screenrc.grml requires files from grml-scripts-core, so I wouldn't want to recommend using that as is.

Reply:
Good point. But AFAICS it's [/etc/grml/screenrc](https://raw.githubusercontent.com/grml/grml-etc-core/master/etc/grml/screenrc) which uses the scripts from grml-scripts-core? :thinking:

Reply:
yeah, thats exactly that...

> RewriteRule screenrc.grml(/?)$ https://git.grml.org/f/grml-etc-core/etc/grml/screenrc